kernel/packages/decentraland-ecs/types/@decentraland/PortableExperiences/index.d.ts

```diff
@@ -0,0 +1,26 @@
+declare module '@decentraland/PortableExperiences' {
+  enum ExecutorType {
+    SCENE = 'SCENE',
+    WEARABLE = 'WEARABLE',
+    QUEST_UI = 'QUEST_UI'
+  }
+  type Executor = {
+    type: ExecutorType
+    identifier: string
+  }
+  type ContentIdentifier = string
+  type PortableExperienceIdentifier = string
+  type PortableExperienceHandle = {
+    pid: string
+    identifier: PortableExperienceIdentifier
+    parentProcess: Executor
+  }
+  type SpawnPortableExperienceParameters = {
+    contentIdentifier: ContentIdentifier
```

Comment:
We should use URN here

```suggestion
    urn: ContentIdentifier
```